### PR TITLE
[minor] add AUS QLD

### PR DESCRIPTION
**Pull request by camjc**
Annoyingly these are separate pages per day, and then just in a paragraph.

### CODE REVIEW

File: scrapers/AUS/QLD/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const scraper = {
+  country: 'AUS',
+  state: 'Queensland',
+  url: 'https://www.health.qld.gov.au/news-events/doh-media-releases',
+  type: 'paragraph',
+  priority: 1,
+  async scraper() {
+    const $ = await fetch.page(this.url);
+    const urls = $('#content h3:first-of-type > a');
+    const currentArticleUrl = urls[0].attribs.href;
```
Comment (camjc) [inline scrapers/AUS/QLD/index.js]:
```suggestion
    const anchors = $('#content h3:first-of-type > a');
    const currentArticleUrl = anchors[0].attribs.href;
```

**Comment by hyperknot** [comment]
What does `type: 'paragraph'` do? Just an information for the scoring system?

**Comment by camjc** [comment]
Yeah I thought that was one of the options instead of table. Might have got the enumeration wrong?

**Comment by lazd** [comment]
Yes, `type: 'paragraph'` is information for rating system.

File: scrapers/AUS/QLD/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  priority: 1,
+  async scraper() {
+    const $ = await fetch.page(this.url);
+    const anchors = $('#content h3:first-of-type > a');
+    const currentArticleUrl = anchors[0].attribs.href;
+    const $currentArticlePage = await fetch.page(currentArticleUrl);
+    const paragraph = $currentArticlePage('#content h2:first-of-type + p').text();
+    const { casesString } = paragraph.match(/state total to (?<casesString>\d+)./).groups;
+    const cases = parse.number(casesString);
+    return [{ state: scraper.state, cases }];
```
Comment (lazd) [inline scrapers/AUS/QLD/index.js]: Note you can return an object here and it'll work fine.

**Pull request merged**